backend/src/main/java/com/googleinterns/smb/CartDocumentsController.java

```diff
@@ -0,0 +1,49 @@
+package com.googleinterns.smb;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import org.springframework.web.bind.annotation.*;
+
+class ItemDetails{
+	public String userID;
+	public String itemName;
+	public int quantity = 0;
```

Comment:
*int* is default to zero, so no initialisation required.
As a good practice, all the field should be **private** and can have **public** getter setters